pkg/controller/worker/machines.go

```diff
@@ -296,8 +305,8 @@ func createDiskSpec(size string, boot bool, machineImage, volumeType *string, wo
 		disk["labels"] = labels
 	}
 
-	if machineImage != nil {
-		disk["image"] = *machineImage
+	if image != nil {
```

Comment:
Do we need any changes in `machine-controller-manager-provider-gcp` or just setting `image` is good enough ? Just wondering why we decided to choose `dataVolume.sourceImage` instead of simply `dataVolume.image` ?

Reply:
yes, nothing to change in the mcm.
we went for `sourceImage` because that's how it is referred to by gcp client.